admin/js/produk.js

```diff
@@ -0,0 +1,232 @@
+var table = document.getElementById("tabel");
+var rows = table.rows;
+for (var i = 1; i < rows.length; i++) {
+    rows[i].onclick = (function (e) {
+        // $("#id").val(this.cells[0].innerHTML);
+        // $("#nama_produk").val(this.cells[1].innerHTML);
+        // $("#jenis_produk").val(this.cells[2].innerHTML);
+        // $("#kategori").val(this.cells[3].innerHTML);
+        // $("#gambar").val(this.cells[4].innerHTML);
+        // $("#video").val(this.cells[5].innerHTML);
+        // $("#spek").val(this.cells[6].innerHTML);
+        // $("#form").attr("action","system/edit-produk.php?page=produk");
+        // $("#delete").attr("href","system/delete.php?page=produk&id="+this.cells[0].innerHTML);
+        var id_temp = this.cells[0].innerHTML;
+        $("#edit-modal").load("page/ajax-edit-produk.php", {id:id_temp}, function (response, status, request) {
+            this; // dom element
+        });
+        $("#delete").attr("href", "system/delete_produk.php?id="+id_temp);
+        $("#form_edit").attr("action", "system/edit_produk.php?id="+id_temp);
+    });
+}
+
+
+var multiple_gambar = 1; //----variabel untuk gambar multi---
+function tambah_image() { 
+    multiple_gambar++;
+    $("#jumlah_gambar").val(multiple_gambar);
+    $("#gambar-grup").html($("#gambar-grup").html()+"<input type='file' name='gambar"+multiple_gambar+"' id='gambar-multi"+multiple_gambar+"' ><button type='button' id='close-btn"+multiple_gambar+"' class='btn btn-danger btn-sm' data-target='"+multiple_gambar+"' onclick='close_btn(this)'><i class='fas fa-minus'></i></button>");
+}
+
+function close_btn(elem) { 
+    multiple_gambar--;
+    $("#jumlah_gambar").val(multiple_gambar);
+    $("#gambar-multi"+$(elem).attr("data-target")).remove();
+    $(elem).remove();
+}
+
+function add_gambar_multi(){
+    $("#form-gambar").html("<div class='row' id='gambar-form-multi'><div class='col-md-10'><label for='gambar-multi'>Gambar</label><div id='gambar-grup'><input type='file' name='gambar1' id='gambar-multi' class='form-control-file ml-auto' required></div></div><div class='col-md-2'><button type='button' class='btn btn-primary' id='tambah-image' onclick='tambah_image()'><i class='fas fa-plus'></i></button></div></div><input type='hidden' name='jumlah_gambar' id='jumlah_gambar' value=1>");
+    multiple_gambar = 1;
+}
+
+function add_specification() {
+    $("#form-specification").html("Specification<div id='toolbar-container'></div><div id='editor' class='border-gray'></div><input type='hidden' name='spek' id='specification'>");
+    DecoupledEditor
```

Comment:
Codacy found an issue: ['DecoupledEditor' is not defined.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)